src/TSMapEditor/UI/TopBar/TopBarMenu.cs

```diff
@@ -127,6 +133,22 @@ public override void Initialize()
                 }
             }
 
+            var theaterMatchingCliffs = map.EditorConfig.Cliffs.Where(cliff => cliff.AllowedTheaters.Exists(
+                theaterName => theaterName.Equals(map.TheaterName, StringComparison.OrdinalIgnoreCase))).ToList();
+            int cliffCount = theaterMatchingCliffs.Count;
+            if (cliffCount > 0)
+            {
+                if (cliffCount == 1)
+                {
+                    editContextMenu.AddItem("Draw Cliff", () => mapView.EditorState.CursorAction =
+                        new DrawCliffCursorAction(mapView, theaterMatchingCliffs[0]), null, null, null);
+                }
+                else
+                {
+                    editContextMenu.AddItem("Draw Cliff...", SelectCliff, null, null, null);
```

Comment:
```suggestion
                    editContextMenu.AddItem("Draw Cliff...", () => selectCliffWindow.Open(), null, null, null);
```

Then the `SelectCliff` method can be removed.